fpl/fpl.py

```diff
@@ -91,19 +101,19 @@ def __init__(self, session):
         :type return_json: bool
         :rtype: list
         """
-        url = API_URLS["static"]
-        teams = await fetch(self.session, url)
-        teams = teams["teams"]
+        teams = getattr(self, "teams")
 
         if team_ids:
             team_ids = set(team_ids)
-            teams = [team for team in teams if team["id"] in team_ids]
+            teams = [team for team in teams.values() if team["id"] in team_ids]
+        else:
+            teams = [team for team in teams.values()]
 
         if return_json:
             return teams
 
-        return [Team(team_information, self.session)
-                for team_information in teams]
+        return {team_information["id"]: Team(team_information, self.session)
```

Comment:
As I said I'm hesitant to add this as it will break people's code, including mine, if they upgrade to a new version. Idk what a solution for this is.